dwellingly/src/views/dashboard.js

```diff
@@ -133,8 +133,8 @@ export class Dashboard extends React.Component {
                     </div>
```

Comment:
We'll need to delete lines 47-54 of this file:
`<div className="dashboard__sidebar_container">
                        <div className="dashboard__sidebar_user_container">
                            <span className="icon dashboard__sidebar_user_icon">
                                <i className="fas fa-user"></i>
                            </span>
                            <span className="dashboard__sidebar_user_name">Raymond Holt</span>
                        </div>
                    </div>`

so that we don't see two left nav containers